interop_binaries/src/lib.rs

```diff
@@ -117,7 +117,7 @@ pub enum VdafObject {
         length: NumberAsString<usize>,
     },
     Prio3Histogram {
-        buckets: Vec<NumberAsString<u64>>,
+        buckets: NumberAsString<usize>,
```

Comment:
I used `length` instead of `buckets` in divergentdave/draft-dcook-ppm-dap-interop-test-design#41, again to align with https://cfrg.github.io/draft-irtf-cfrg-vdaf/draft-irtf-cfrg-vdaf.html#section-7.4.3-2.